app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+    Train to teach in England as a non-UK citizen : "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+    Teach in England if you trained outside the UK: "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
+    Apply for your visa to teach in England: "/non-uk-teachers/visas-for-non-uk-teachers"
+    Apply for your visa to train to teach in England: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees"
+
+promo_content:
+date: "2021-05-27"
+image: "static/content/hero-images/0002.jpg"
+backlink: "../../"
+keywords:
+  - International
+  - Overseas
+  - Foreign
+  - Nationality
+  - Residency
+  - EEA
+  - Canada
+  - New Zealand
+  - Switzerland
+  - USA
+  - Visa
+  - Skilled worker visa
+  - EU
+  - EU Settlement Scheme
+  - NARIC
+  - UK NARIC
+  - ENIC
+  - UK ENIC
+  - European Network of Information Centre
+  - National Recognition Information Centre for the United Kingdom
+  - PhD
+  - Immigration
+  - Ireland
+  - Irish
+  - Common Travel Area
+  - Student Visa
+  - Home Office
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Japan
+  - Monaco
+  - Hong Kong
+  - Republic of Korea
+  - Taiwan
+  - Family Visa
+  - UK Ancestry Visa
+  - Abroad
+  - Europe
+  - European
+  - Settlement Scheme
+---
+
+## Who is eligible
+
+The information on this page is for: 
+
+* non-UK teachers of languages and physics 
+* non-UK trainee teachers of languages and physics   
+
+If you are considering coming to England to teach or train to teach languages or physics, you could be eligible for financial help from the UK government worth £10,000.  
+
+The international relocation payment (IRP) is designed to repay some of the costs of moving to England, including: 
+
+* the costs of visas 
+
+* the immigration health surcharge 
+
+* other relocation expenses 
+
+If you are eligible, you’ll receive a single payment of £10,000 from the Department for Education (DfE) around the end of your first term training or working in England. You will not have to pay this money back. You can only receive the IRP once. 
+
+## How to apply 
+
+The process of applying for the international relocation payment is different for trainee teachers and teachers.  
+ 
+## Criteria for trainee teachers  
+
+As a trainee teacher, to be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.
+
+### 1. Teaching subject
+ 
+You must have taken up a place on a teacher training course in England leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) for the academic year 2023 to 2024. Your teacher training provider must be accredited by the UK government, and you must be training to teach one of the following: 
+
+a) any secondary language or languages, including courses with content which is at least 50% languages. You can combine training to teach English with another language, as long as the non-English language makes up at least 50% of your course. 
```

Comment:
This needs to be coded as a list from an accessibility point of view

Reply:
Ah really OK cool